### ISSUE TITLE
About Training

### ISSUE THREAD

**Issue by jhj7905** [YadiraF/PRNet#118]
@YadiraF @developer-mayuan @nilday

Hello
I have a question about train
When i trained the network, the outputs of prediction are same as input(position map) like below
![temp](https://user-images.githubusercontent.com/21155309/55299627-b74e2b80-546e-11e9-830a-aa9de0fca7eb.PNG)
i thought the problem was related to loss function(i mean 'weight')
Parts of my code like below
`uv_weight_mask = cv2.imread('/home/hyunjo/3D_Mesh/prnet_train/PRNet/Data/uv-data/uv_face_mask.png')
uv_weight_mask = cv2.cvtColor(uv_weight_mask, cv2.COLOR_BGR2RGB)
uv_array = np.array(uv_weight_mask, dtype=np.float32)
uv_array = uv_array/255.0

x = tf.placeholder(tf.float32, shape=[None, 256, 256, 3])
label = tf.placeholder(tf.float32, shape=[None, 256, 256, 3])
weight = tf.placeholder(tf.float32, shape=[None, 256, 256, 3])

net = resfcn256(256, 256)
x_op = net(x, is_training=True)
loss = tf.losses.mean_squared_error(label, x_op, weights=weight)`
Could you tell me advice?
Thank you in advance

**Comment by yexiaoxi01** [comment]
@jhj7905  Hello，I am very similar to your training, with the same weight. There are also some problems with the results. Did you solved this problem? My image and model do not matched.like this:
![image](https://user-images.githubusercontent.com/47855945/56357593-2077c200-6217-11e9-851f-b75df25b678e.png)
![image](https://user-images.githubusercontent.com/47855945/56357628-35ecec00-6217-11e9-9ea5-ea5341dcfd52.png)

**Comment by leowang7** [comment]
Hi jhj7905，I was traning the PRnet at the coment, just came across your questions. I wonder your posiiton maps shown above are in BGR not in RGB channels.

**Comment by qiannvMao** [comment]
@jhj7905 @yexiaoxi01 Hi,as a newer to this, I failed to achieve the training code, could you please provide training code to me? I hope I can get your help, thank you very much. My email: mqn20170910@163.com

**Comment by yexiaoxi01** [comment]
@qiannvMao  i just try the traing code provide by https://github.com/YadiraF/PRNet/issues/46#issuecomment-455841527

**Comment by qiannvMao** [comment]
@yexiaoxi01 thank you, I will  try.
